client/src/app/panel/tabs/linting/LintingTab.js

```diff
@@ -124,6 +130,13 @@ function LintingTabItem(props) {
     </div>
     <div className="linting-tab-item__content">
       { message }
+      {
+        !isNil(documentationUrl) && <>
+          <a className="linting-tab-item__link" href={ documentationUrl } target="_blank" rel="noopener noreferrer" onClick={ stopPropagation }>
```

Comment:
Can we add a descriptive title here such as "Open rule documentation"?

Reply:
👍🏻

Reply:
https://github.com/camunda/camunda-modeler/pull/3713/commits/9d5951d68f2e8049debe831504d0c5a5684330ec 👀